Support/lib/validate_on_save/validators/ruby.rb

```diff
@@ -22,9 +22,16 @@ def self.ruby
     def self.rubocop
       rubocop_bin = ENV['TM_RUBOCOP'] || "rubocop"
       filepath = ENV["TM_FILEPATH"]
+
+      info = `echo Running syntax check with rubocop-$("#{rubocop_bin}" --version)`
+
+      format = VOS.opt("VOS_HTML") ? "emacs" : "clang"
+      result = `"#{rubocop_bin}" --format #{format} "#{filepath}"`
+      result = VOS.htmlify_results(result, filepath)
```

Comment:
maybe delete the  `return result unless VOS.opt("VOS_HTML")` from the `htmlify_results` method to the validation (RuboCop, JSONLint and Shellcheck) files, like:
```
      result = VOS.htmlify_results(result, filepath) if VOS.opt("VOS_HTML")
```